website/docs/r/linux_function_app.html.markdown

```diff
@@ -139,6 +139,11 @@ The following arguments are supported:
 
 ~> **Note:** Assigning the `virtual_network_subnet_id` property requires [RBAC permissions on the subnet](https://docs.microsoft.com/en-us/azure/app-service/overview-vnet-integration#permissions)
 
+[//]: # (todo: add it in 4.0 provider)
```

Comment:
Same with the comment above, can we make this 
```suggestion
[//]: # (TODO 4.0 add it in 4.0 provider)
```
So it's consistent and easier to find